pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -180,6 +180,25 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 		},
 	}
 
+	if options.DNSConfig != nil {
+		_, dnsPortStr, err := net.SplitHostPort(options.DNSConfig.BindAddress)
+		if err != nil {
+			return nil, fmt.Errorf("unable to parse DNSConfig.BindAddress: %v", err)
```

Comment:
`DNSConfig.BindAddress` won't be meaningful to them, since that's not what the config field is named. What about `unable to parse DNS bind address %s: %v", options.DNSConfig.BindAddress, err)`